charts/jenkins/values.yaml

```diff
@@ -3,7 +3,7 @@ clusterAdminEnabled: false
 jenkins:
   rbac:
     create: true
-    readSecrets: false
+    readSecrets: true
```

Comment:
why?

Reply:
to allow the kubernetes-credentials-provider plugin to read secrets and turn them into jenkins credentials

Reply:
What's the reason to no use the credentials defined from the configuration as code plugin? 
Several months ago we stopped using kubernetes secrets in favor of jcasc

Reply:
This gives us the option of using kubernetes external secrets with the credentials provider to load them directly from a cloud hosted secrets manager (rather than using the sops option)

Reply:
I've not moved any secrets or changed how it currently works, just enabled the plugin so we have the option.  Using this approach secret rotation becomes far easier.